health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewChecklist.js

```
@@ -0,0 +1,240 @@
+        },
+        params: {
+            locale: "en_MZ",
+            tenantId: tenantId,
+            module: "hcm-campaignmanager"
+        },
+    }
+    const { isLoading2, data: localization, isFetching2 } = Digit.Hooks.useCustomAPIHook(reqCriteria);
+
+    useEffect(() => {
+        if (localization?.messages?.length > 0) {
+            let matchedItem = localization.messages.find(item => item.message === checklistType);
```

Biome

[error] This let declares a variable that is only assigned once.

'matchedItem' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)